### PR TITLE
Update configure-advanced-scan-types-microsoft-defender-antivirus.md

### LINKED ISSUE #12354: about setting value "Scan packed executables"
[Enter feedback here]
"Scan packed executables" has been removed from the following Windows 11 templates

Administrative Templates (.admx) for Windows 11 2022 Update (22H2)
Administrative Templates (.admx) for Windows 11 October 2021 Update (21H2)

Could you please remove it from this site as well or add a comment.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 292d5bf4-ff14-4cbe-eab1-3fe35cb5c1c9
* Version Independent ID: 782db2b2-0a15-941c-24ff-345ecfe694f3
* Content: [Configure scanning options for Microsoft Defender Antivirus](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/configure-advanced-scan-types-microsoft-defender-antivirus?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/configure-advanced-scan-types-microsoft-defender-antivirus.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/configure-advanced-scan-types-microsoft-defender-antivirus.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

### PR DESCRIPTION
**Pull request by msbemba**
added a note about availability of scan packed executables in windows 11

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12354

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/configure-advanced-scan-types-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -75,6 +75,11 @@ For details on configuring Microsoft Configuration Manager (current branch), see
 |Specify the maximum size (in kilobytes) of archive files that should be scanned. <p> **Scan** \> **Specify the maximum size of archive files to be scanned**|No limit|Not available <p>The default value of 0 applies no limit|
 |Configure low CPU priority for scheduled scans <p> **Scan** \> **Configure low CPU priority for scheduled scans**|Disabled|Not available|
 
+> [!NOTE]
+> Scan packed executables have been removed from the below templates
+> 1) Administrative Templates (.admx) for Windows 11 2022 Update (22H2)
+> 2) Administrative Templates (.admx) for Windows 11 October 2021 Update (21H2)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-advanced-scan-types-microsoft-defender-antivirus.md]:
```suggestion
> Scan packed executables have been removed from the following templates:
> - Administrative Templates (.admx) for Windows 11 2022 Update (22H2)
> - Administrative Templates (.admx) for Windows 11 October 2021 Update (21H2)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**